src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.kt

```diff
@@ -304,6 +304,11 @@ class DetectProperties {
             setHelp("If set to true, only Detector's capable of running without a build will be run.")
             setGroups(DetectGroup.GENERAL, DetectGroup.GLOBAL)
         }
+        val DETECT_CARGO_PATH = NullablePathProperty("detect.cargo.path").apply {
```

Comment:
Why are we taking in the cargo executable? It doesn't look like you use it anywhere.

Reply:
yeah I'm really not sure why I did that